requests/constants.js

```diff
@@ -0,0 +1,26 @@
+const DEV_FEATURE_FLAG = true;
+const Status = {
+  APPROVED: 'APPROVED',
+  PENDING: 'PENDING',
+  REJECTED: 'REJECTED',
+};
+
+const LAST_ELEMENT_CONTAINER = '.virtual';
+const REQUEST_ACTION_BUTTON = 'request-action-button';
+const SEARCH_BUTTON_ID = 'search_button';
+const SORT_DROPDOWN_ID = 'sort_by';
+const STATUS_DROPDOWN_ID = 'status';
+const OOO_CONTAINER_ID = 'ooo-request-container';
```

Comment:
Can we please follow one naming convention?

Reply:
changes done, please check.